README.md

```diff
@@ -104,7 +104,7 @@ storIOSQLite
     .table("tweets")
     .build())
   .prepare()
-  .createObservable() // Get Result as rx.Observable and subscribe to further updates of tables from Query!
+  .asRxObservable() // Get Result as rx.Observable and subscribe to further updates of tables from Query!
```

Comment:
Lol, here we literaly saying `Get Result as rx.Observable` :smile: 

So, I guess, new method name is okay :)